articles/multifactor-authentication/developer/index.md

```diff
@@ -0,0 +1,29 @@
+---
+toc_title: Developer Documentation for Guardian
+url: /multifactor-authentication/developer
+description: Developer Documentation for Guardian
+---
+
+# Developer Resources for Guardian
+
+At Auth0, we believe in giving developers the tools they need to get their jobs done. With Guardian, developers are able to customize their users' experience, and even build their own applications on top of our multifactor capabilities.
+
+## Getting started with Guardian within your Application
+Most often, administrators will manage multifactor configuration for an application, as seen [here](multifactor-authentication/guardian/admin-guide). For more advanced use cases, Guardian provides developers the ability to control user access directly.
+* [Example Application using Guardian for Multifactor Authentication](https://github.com/auth0/guardian-example)
+* [How to do 'Step-Up' Authentication with Guardian](/multifactor-authentication/developer/step-up-with-acr)
+
+## Customizing the Guardian Widget
+Apply the look-and-feel of your organization by working with our Guardian client libraries.
+* [Client library for Auth0 Guardian](https://github.com/auth0/auth0-guardian.js)
+* [Creating a Custom Guardian Widget](https://github.com/auth0/auth0-guardian.js/tree/master/example)
+
+## Working with Guardian within your Application
+Most often, administrators will manage multifactor configuration for an application, as seen [here](multifactor-authentication/guardian/admin-guide). For more advanced use cases, Guardian provides developers the ability to control user access directly.
+* [Example Application using Guardian for Multifactor Authentication](https://github.com/auth0/guardian-example)
+* [How to do 'Step-Up' Authentication with Guardian](/multifactor-authentication/developer/step-up-with-acr)
```

Comment:
Both links under paragraphs `Getting started with Guardian within your Application` and `Working with Guardian within your Application` are identical. The text before the links is also exactly the same so I'm guessing it's a copy paste error.